### ISSUE TITLE
PCjr/Tandy review

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#4398]
As per post in discord by AlphaHex:

I did review all PCjr/Tandy games on this comprehensive listing https://docs.google.com/spreadsheets/d/17LFchbehz7WQw1pcl0ae3D9ZRx8xahuH/edit#gid=612489937

I have reviewed also eXoDOS github to eliminate duplicates
here is my summary

Missing (available in TDC)

Big Bird's Special Delivery v1.5 (1992)(Hi-Tech Expressions, Inc.)
Peanuts Picture Puzzlers (1984)(Random House, Inc.) [Educational]
Time Bound (PCjr) (1984)(CBS Software) [Action, Educational]
Music 101- Trivia from Mozart to Motown (1985)(IBM) [Educational, Trivia]
Trivia 103 (1985)(IBM) [Educational, Trivia]
Kidwriter (1984)(Spinnaker Software Corporation)

Improvements

Demon Attack (1984)(Tandy Corporation) [IMA][360k] - has title screen which PCjr does not have
Trivia 101- The Introductory Course (1984)(IBM) [Educational, Trivia] -> take from TDC as eXoDOS has wrong image file
TV and Cinema 101: Trivia from Talkies -> switch to ece or staging for working PCjr support
Duck Tales : Quest for the Gold -> machine should be set to Tandy for better music
Sea Hunt -> machine should be set to PCjr for better music
Snow Strike -> machine should be set to Tandy for better music
Trains -> machine should be set to Tandy for better music

Newer versions (available in TDC)

GATO v1.4 [f1] (1984)(Spectrum HoloByte, Inc.) [Simulation]
Jet v1.3 (1985)(subLOGIC) [Simulation]
Jet v2.12 (1988)(subLOGIC) [Simulation]

Enhancements

set all PCjr, Tandy and CGA composite games to DOSBox Staging so we can play with composite settings via F12/F11

Metadata

Transylvania - release on date is 1982 should be 1986

**Comment by Python-Exoproject** [comment]
Further to that I have gone over the suggestions and add:

Missing

Big Bird's Special Delivery v1.5 (1992)(Hi-Tech Expressions, Inc.)
- We have a copy of this in exodos but looking at the source doc I think he meant to say that we have the CGA Hi-Tech Expressions re-release and are missing the original PCjr CBS Software release

Ernie's Magic Shapes
- Not on his list as I dont think in TDC but same as above

Trivia 102
- Not on his list as I dont think in TDC but as others of series are mentioned Ill add it here. I assume Trivia 102 and Trivia 103 along with another in his missing list, Music 101, should all be added as PCjr if they are found going by the improvement noted in Trivia 101 and TV and Cinema 101.


Improvements

Demon Attack (1984)(Tandy Corporation) [IMA][360k] - has title screen which PCjr does not have
- Above true according to youtube but our copy is the pcjr release, and I couldnt find a copy of the Tandy release for test

Trivia 101- The Introductory Course (1984)(IBM) [Educational, Trivia] -> take from TDC as eXoDOS has wrong image file
- I didnt see a difference between the two images when booted? Should be switched to staging or ECE either way though with pcjr as it gives improved sound

Sea Hunt -> machine should be set to PCjr for better music
- Not just music, game was definitely meant to be PCjr as now the colour selection has an effect when in game vs currently where no matter your choice the game board shows in CGA

Trains -> machine should be set to Tandy for better music
- There is actually currently no music, so tandy is big improvement


Additional

Pinball Construction Set (1985)
- Switch to Tandy for better colours

**Comment by Python-Exoproject** [comment]
I also agree should consider moving all tandy\pcjr off vanilla onto ece or staging. 

Example reason is colours can be wrong for PCjr in vanilla. For "Webster - The Word Game (1984)" the title is impossible to read on the initial screen under vanilla. Moving to ECE or Staging uses the correct colours and you can then see the title. 

Another reason is sound accuracy as ECE has improved sound emulation over vanilla, and staging have taken it another step further.

Lastly, as AlphaHex pointed out it would allow these games to be switched to composite by the user for any that we missed if desired which is not possible using vanilla.

**Comment by exoscoriae** [comment]
moved all PCjr games to ece, except for Boppie which currently uses TC_Dosbox
Fixed metadata for Transylvania 2. 
All "Improvements" made
Pinball Construction Set (1985) moved to tandy

**Comment by exoscoriae** [comment]
not having the big bird pcjr release means.... well, we can't do anything about it.  Best we can do is add it as a new ticket under game submissions and assign it the "MISSING" tag

**Comment by exoscoriae** [comment]
other than the missing titles, the last actionable thing I see here is moving machine=tandy games to staging\ece

**Comment by Python-Exoproject** [comment]
List of games using tandy in exodos pack as at August 2023 for reference: 
[Tandy.txt](https://github.com/exoscoriae/eXoDOS/files/12386286/Tandy.txt)

**Comment by exoscoriae** [comment]
is there anything left to action on this ticket at this point @Python-Exoproject ?

Or do we close it and let the other tickets handle it?

**Comment by Python-Exoproject** [comment]
This ticket is still active in regards to moving games that have a tandy option to Staging as its tandy emulation, especially around tandy sound, is superior.

**Comment by exoscoriae** [comment]
ok... so, is there anything actionable on it currently @Python-Exoproject ?

The most recent post from you just says "deleted list".

So is this on hold until a new list is made?

**Comment by Python-Exoproject** [comment]
Yeah, on hold until list is made. 

Im actually thinking that whatever tool @boohyaka comes up with to move vanilla dosbox games over to Staging could be used here as well. Once all these tickets have been processed and the tool has been tested and proved on the simple vanilla conversion we can list all games that have machine=tandy in conf or exception.bat, machine=pcjr in conf or exception.bat, or tandy=on in conf or run.bat and run it on them as well

Before you release the next update but after youve processed #5045 we can probably get just the list of games that have machine=tandy in conf or exception.bat and create a Tandy playlist for Launchbox. Such a list only needs to have games with machine=Tandy as they are the true Tandy games, ones that use Tandy sound without being machine=tandy arent really Tandy games

In Summary: On hold until we have a list of game running Tandy mode to a) move to Staging where not already doing so, and b) create a Tandy playlist in LB

**Comment by exoscoriae** [comment]
https://github.com/exoscoriae/eXoDOS/issues/5045 has been processed.
